docs/onboarding/README.md

```diff
@@ -12,7 +12,7 @@ These are the steps to onboard your fund to the DCAF-managed DARIA pipeline.
 * Someone from the DARIA team (probably @nerdygirl537 or @sofijaj) will hop on a call with your team to demonstrate DARIA's features and workflow, and to talk through your process to see if this is a good fit for you.
```

Comment:
optional: this is probably you, me, or Lisa now, ya?

Reply:
```suggestion
* Someone from the DARIA team (probably @colinxfleming, @lomky, or @lwaldsc) will hop on a call with your team to demonstrate DARIA's features and workflow, and to talk through your process to see if this is a good fit for you.
```